README.md

```diff
@@ -311,6 +320,29 @@ Note that when using an image customized with `install-gpu`, the hosted docker
 container should be set to run in privileged mode so that it has access to the
 GPU device on the host machine.
 
+#### seal-oem
+
+The `seal-oem` build step utilizes `dm-verity` to verify the data in the OEM
+partition when the system boots. If the verification fails, the system will 
+refuse to boot. This step takes no flags and needs to be run after any step 
```

Comment:
The verification should also occur when we use any binary present in OEM partition and therefore, it there is any change from the expected value a kernel panic will occur.